manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -310,6 +310,20 @@ spec:
       failureThreshold: {{ annotation .ObjectMeta `readiness.status.sidecar.istio.io/failureThreshold` .Values.global.proxy.readinessFailureThreshold }}
     {{ end -}}
     securityContext:
+      {{- if eq (index .ProxyConfig.ProxyMetadata "IPTABLES_TRACE_LOGGING") "true" }}
```

Comment:
Not sure if IPTABLES_TRACE_LOGGING should have all this broad effect.

Maybe have a more precise setting that enables NET_ADMIN/coredump ?